src/Common/TurbulenceClosures/TurbulenceClosures.jl

```diff
@@ -190,13 +198,55 @@ function flux_second_order!(
     t::Real,
 ) end
 function compute_gradient_flux!(
-    h::HyperDiffusion,
+    ::HyperDiffusion,
+    diffusive::Vars,
+    ∇transform::Grad,
+    state::Vars,
+    aux::Vars,
+    t::Real,
+) end
+
+## Default methods for divergence damping models
+function init_aux_divdamping!(
+    ::DivergenceDampingModel,
+    ::BalanceLaw,
+    aux::Vars,
+    geom::LocalGeometry,
+) end
+function atmos_nodal_update_auxiliary_state!(
```

Comment:
```suggestion
function nodal_update_auxiliary_state!(
```

Reply:
or `hyperdiffusion_nodal_update_auxiliary_state!`. We should really try to standardize these names..